routers/player.py

```diff
@@ -1,11 +1,56 @@
 from fastapi import APIRouter, Depends
 from sqlalchemy.orm import Session
+from sqlalchemy import select, text
 from db import get_db
 from models.player import Player
+from schemas.player import PlayerSchema
 
 router = APIRouter()
 
-@router.get("/player")
-def get_player(db: Session = Depends(get_db)):
-    players = db.query(Player).all()
-    return players
\ No newline at end of file
+
+@router.get("/player") 
+async def get_player(db: Session = Depends(get_db)):
+    sql = """
+    SELECT * FROM player
+    """
+
+    result = await db.execute(text(sql))
+    player = result.fetchone()
+
+    return player._asdict()
+
+@router.post("/player")
+async def create_player(player: PlayerSchema, db: Session = Depends(get_db)):
+    sql = """
+    INSERT INTO player (id, money)
+    VALUES (id, money)
+    """
+
+ 
+    await db.execute(text(sql), {"id": player.id, "money": player.money})
+    await db.commit()
+
+    sql_select = """
+    SELECT * FROM player WHERE id = :id
+    """
+    result = await db.execute(text(sql_select), {"id": player.id})
+    new_player = result.fetchone()
+
+    return dict(new_player._mapping) if new_player else None
+
+@router.put("/player/money")
+async def update_money(player_id: int, player: PlayerSchema, db: Session = Depends(get_db)):
```

Comment:
```suggestion
async def update_money(player: PlayerSchema, db: Session = Depends(get_db)):
```
player_idは消しちゃってもいいかも